### ISSUE TITLE
Sass import error when using most recent version

### ISSUE THREAD

**Issue by JGrishey** [GetStream/stream-chat-react#1047]
Hey all!

I seem to be getting an import error when using this package:

```
Error: File to import not found or unreadable: ../../node_modules/mml-react/src/styles/index.scss.
          on line 7:1 of node_modules/stream-chat-react/dist/scss/index.scss
          from line 25:1 of app/assets/stylesheets/chat.scss
  >> @import '../../node_modules/mml-react/src/styles/index.scss';
```

If I change the `@import` line to include an extra backwards in the directory, like:

```
@import '../../../node_modules/mml-react/src/styles/index.scss'
```

Then it compiles fine. Is anyone else seeing this, or is this a problem only with our project's configuration?

**Comment by DanC5** [comment]
Hello @JGrishey I believe this is happening because `mml-react` is an optional, lazy-loaded dependency. So if you never utilize the `MML` component, you'll never install the dependency. I now see where this could become an issue, importing some scss that won't always be there. I'll look into a way to conditionally import the scss too.

Is this affecting your ability to use the library at all? Or were you just more curious about the error?

**Comment by DanC5** [comment]
Just for reference, I'm running two apps at the moment that don't use `MML`, so that path doesn't exist in either app's node modules, but they both run fine even with the bad import.

**Comment by DanC5** [comment]
@JGrishey hopefully the CSS bundling issues are fixed in the latest release, https://github.com/GetStream/stream-chat-react/releases/tag/v6.4.4. Let me know if this solves the issue for you, thanks.

**Comment by DanC5** [comment]
@JGrishey I've received reports that this issue is fixed in 6.4.4 so I'm going to close, but do reach out if you need additional assistance, thanks!
